### DISCUSSION TITLE
Add an option to control whether to use rich progress bar in BaseInference 为 BaseInferencer 加一个开关，用于控制是否开启进度条

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#265] [MMEngine]
# Add an option to control whether to use rich progress bar in BaseInference 
 为 BaseInferencer 加一个开关，用于控制是否开启进度条 

Add an option to control whether to use rich progress bar in [BaseInference](BaseInference) 
为 BaseInferencer 加一个开关，用于控制是否开启进度条
具体请见：[https://github.com/open-mmlab/mmengine/issues/1112](https://github.com/open-mmlab/mmengine/issues/1112) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Easy

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1135

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by CastleDream** [comment]
This task has been merged in [[Enhancement] Add an option to control whether to show progress bar in BaseInference #1135](https://github.com/open-mmlab/mmengine/pull/1135).
Please close this task.

**Reply by crazysteeaam** [reply]
done~
